cve_bin_tool/parsers/parse.py

```diff
@@ -7,38 +7,18 @@
 from cve_bin_tool.parsers.r import RParser
 from cve_bin_tool.parsers.rust import RustParser
 
-valid_files = [
-    "pom.xml",
-    "package-lock.json",
-    "Cargo.lock",
-    "renv.lock",
-    "PKG-INFO: ",
-    "METADATA: ",
-]
+valid_files = {
```

Comment:
A thought with this nice list: should we be outputting the list of supported files/languages the same way we report the available checkers? That might be a nice feature to add.  I'll open a separate issue, though, since this otherwise looks ready to merge.